server/services/licenceService.ts

```diff
@@ -159,7 +159,9 @@ export default class LicenceService {
           return {
             code: conditionCode,
             sequence: index,
-            category: getAdditionalConditionByCode(conditionCode)?.category,
+            category:
+              getAdditionalConditionByCode(conditionCode)?.categoryShort ||
+              getAdditionalConditionByCode(conditionCode)?.category,
             text: getAdditionalConditionByCode(conditionCode)?.text,
           }
```

Comment:
For conditions which define  a shorter name, this is what we send to the API to store. Its only used on summary screens, and these all use the short description anyway. The conditions select page gets the descriptions from config so always uses the long description.